src/rgw/services/svc_bi_rados.cc

```diff
@@ -125,23 +125,18 @@ static void get_bucket_index_objects(const string& bucket_oid_base,
     if (shard_id < 0) {
       for (uint32_t i = 0; i < num_shards; ++i) {
         if (gen_id) {
-          if (gen_id != 0) {
```

Comment:
Shouldn't this commit be squashed? It's basically a fixup.

Reply:
done